### DISCUSSION TITLE
Alguém já usou ACSS (Atomic CSS) ou Tachyons?

### DISCUSSION THREAD

**Discussion by tiagosatur** [frontendbr/forum#876] [Discussão]
Olá galera!

Li um artigo chamado [How to write better CSS in teams with ACSS — A dynamic Atomic CSS library](https://medium.freecodecamp.org/acss-a-dynamic-atomic-css-library-402dff9756e0) no FreecodeCamp e fiquei curioso.

Um dos grandes problemas que ele se propõe a resolver é a criação de nomes de classes pra todos os elementos HTML. Quem utiliza metodologia BEM já sabe como organizar melhor isso, mas...

Alguém aí já usou o tachyons.io ou o novo ACSS (atomic css -  acss.io) e teria uma opinião quanto a rapidez de desenvolvimento, trabalho em equipe, manutenção do código?

Valeuu!

**Comment by samuelyeshua** [comment]
Faz pouco tempo que estou usando tachyons, até o momento estou gostando bastante dele, com toda certeza o desenvolvimento ficou mais rápido pra mim.

**Comment by richardaum** [comment]
Acabei de ler um artigo interessante. 
https://marcelosomers.com/writing/rationalizing-functional-css/

Ele elencava alguns pontos negativos da abordagem:
- Losing the Cascade
- Reusability of Components
- Responsive
- Managing
- Standard Class Order
